src/test/utils/wslpath.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as assert from 'assert';
+
+import * as wslpath from '../../utils/wslpath';
+
+function mkConverterCheck(converter: (arg: string) => string) {
+  return (from: string, to: string) => assert.equal(converter(from), to);
+}
+
+suite('WSL path conversion', () => {
```

Comment:
@yisonPylkita seeing as you're more knowledgeable about the WSL path conversion, could you please add more tests to better cover the existing/desired behaviour? (It'd also might be a good idea to include tests when making changes like https://github.com/rust-lang/rls-vscode/issues/550)